config/crd/kustomization.yaml

```diff
@@ -34,6 +38,8 @@ patchesStrategicMerge:
 - patches/cainjection_in_openstacknets.yaml
 #- patches/cainjection_in_openstackipsets.yaml
 #- patches/cainjection_in_openstackpredictableips.yaml
+#- patches/cainjection_in_openstackplaybookgenerators.yaml
+#- patches/cainjection_in_openstackephemeralheats.yaml
```

Comment:
We might need to uncomment this line for its associated mutating webhook.  But then again, maybe not if it's already working.

Reply:
I will uncomment and try it. It is working when I last tested deploying the operator via CSV. But maybe there is some magic happening. Also, I haven't tried the 'make run' method.